### DISCUSSION TITLE
Method `cancel` not available

### DISCUSSION THREAD

**Discussion by Cyrille-Dakhlia** [llfbandit/record#239] [Q&A]
Hello 👋 

First of all, thank you very much for this package 🙏 
I have read in the documentation the existence of a `cancel` method to remove the audio, but I can't find it in. Has it been removed?

```dart
import 'package:record/record.dart';

final record = AudioRecorder();

...

// Stop recording...
final path = await record.stop();
// ... or cancel it (and implicitly remove file/blob).
await record.cancel();

record.dispose(); // As always, don't forget this one.
```

**Comment by llfbandit** [comment]
Thanks for the report.
I guess it has never been exposed... 😞

**Reply by Cyrille-Dakhlia** [reply]
Oh :o 
Thank you for your answer 🙏 
So I guess it should be released in a later version? (no pressure ^^)
